### ISSUE TITLE
enable accessibility for screenreaders

### ISSUE THREAD

**Issue by vojtapolasek** [sindresorhus/caprine#1312]
Hi, I am blind and I would like to try Caprine on Linux. Currently, Electron requires a special command line option to enable accessibility. In particular, you have to run it with --force-renderer-accessibility. I never worked with npm etc... could you please tell me wherre to put this argument so that Electron honors it?
I cloned your repo, did npm install and npm start --force-renderer accessibility but this did not work.
Thanks for help.

**Comment by sindresorhus** [comment]
You can add `app.commandLine.appendSwitch('force-renderer-accessibility');` to line 57 in `index.ts`.

But, I don't think it's correct to use this flag as, according to https://www.chromium.org/developers/design-documents/accessibility, the need for accessibility should be detected automatically.
